.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Release
 
 env:
   # 🖊️ EDIT to change the image build settings.
-  IMAGE_NAME: exhort-javascript-api
+  IMAGE_NAME: trustify-da-javascript-client
```

Comment:
This registry doesn't exist. We should consider publishing it to the `trustification` registry instead. We have to disable it for now

Reply:
I will remove this all for now, we can add it back later if we decide to do so